digitalpublicgoods/the-global-partnership-on-marine-litter-digital-platform.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "The Global Partnership on Marine Litter Digital Platform",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.unep.org/explore-topics/oceans-seas/what-we-do/addressing-land-based-pollution/global-partnership-marine#:~:text=The%20GPML%20is%20a%20multi,to%20this%20pressing%20global%20issue."
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "Yes",
+    "pltIndependenceDesc": "Some of the software used in developing the functionalities of the platform are not open source. For example ESRI ArcGIS online has been used in the implementation of the data hub component."
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://wedocs.unep.org/bitstream/handle/20.500.11822/34453/UNEP%20GPML%20Digital%20Platform%20Concept%20for%20User%20and%20Partner%20Consultations%20May%202021.pdf"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Unknown",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "The GPML  Digital Platform Privacy  Policy, Personal Data Protection and Terms of Use is framed within the UN Personal  Data Protection and Privacy Principles, the UNEP Access-to-Information Policy and relevant  UN administrative issuances."
+    ],
+    "adherenceSteps": [
+      "https://digital.gpmarinelitter.org/privacy-policy-and-terms-of-use.pdf"
```

Comment:
The privacy policy/ terms-of-use link appears to be unreachable. Kindly check with submitter for the correct one. Thanks.